src/main/java/httpserver/response/note.java

```diff
@@ -0,0 +1,17 @@
+package httpserver.response;
+
+public class note {
```

Comment:
I believe classes should be Pascal case, Note.

Reply:
The test seemingly requires a response in which the class name is lowercase. I can show you during office hours.